exercises/concept/freelancer-rates/freelancer-rates.spec.js

```diff
@@ -1,6 +1,6 @@
 // @ts-check
 
-import { dayRate, monthRate, daysInBudget } from './numbers';
+import { dayRate, monthRate, daysInBudget } from './freelancer-rates';
 
 describe('numbers', () => {
```

Comment:
Should this also be renamed to:

```suggestion
describe('freelancer-rates', () => {
```